app/views/portal/show.rb

```diff
@@ -36,11 +36,8 @@ def head_meta
       end
     end
 
-    def page_title
-      mustache[:page_title] ||= begin
-        title = [display_title, creator_title]
-        CGI.unescapeHTML(title.compact.join(' | ')) + ' - Europeana'
-      end
+    def page_content_heading
+      t('site.search.page-title', default: 'Search Results')
```

Comment:
I don't think we want to replace the show pages with this, 'Search Results' should only be on search results(plural/index) pages not search result(singular/show) pages